refactor_app_component.md

```diff
@@ -1,18 +1,87 @@
-# Refatorando o App Component
+# #13: 🚧 Refatorando o App Component
 
-Vamos realizar uma pequena refatoração. O componente principal do aplicativo não deve ter um HTML tão grande e com toda essa lógica. Deve apenas chamar outro componente que irá lidar com isso.
+Vamos realizar uma pequena refatoração. O `app-root` não deve ter um tempalte tão grande e com toda essa lógica. Deve apenas chamar outro componente que irá lidar com isso.
```

Comment:
tempalte

Reply:
Hi @alvarocneto!
I did the correction, sorry for that :)